src/FairSignalJob.php

```diff
@@ -93,6 +97,23 @@ public function handle()
         }
     }
 
+    public function getQueueTries($queue)
+    {
+        if(!$queue) {
+            $queue = 'default';
+        }
+
+        // get queue tries from fair-queue config
+        $tries = config("fair-queue.queues.{$queue}.tries");
+
+        // sanity check
+        if(!$tries) {
+            $tries = config('fair-queue.queues.default.tries');
+        }
+
+        return $tries;
```

Comment:
can be simplified as
`
return config("fair-queue.queues.{$queue}.tries") ?? config('fair-queue.queues.default.tries') ?? 1;
`

Reply:
Yes but it won't be that readable I think